### ISSUE TITLE
WCAG: Status Messages

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#25]
## Description
Im not sure if we are within this one. Should do a check if our error messages and dynamics are good.

**Intent**

> The intent of this Success Criterion is to make users aware of important changes in content that are not given focus, and to do so in a way that doesn't unnecessarily interrupt their work.

[Source](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html)

## Loadbar
We have a loadbar that is visible sometimes. This component will need the Aria-atributtet role="progressbar"

## Acceptance criteria
- [x] The user is aware of important changes in content
  - [x] Status for validation errors that are activated onBlur. [Look at this article for example ](https://aksel.nav.no/god-praksis/artikler/skjemavalidering?tema=universell-utforming)
  - [x] Check if our loader should have has role="progressbar"

**Comment by FinnurO** [comment]
Would #117 be relevant for this case?

**Comment by framitdavid** [comment]
@FinnurO the issue you are linking to can be beneficial for making sure that users be aware of changes in the content. We should resolve both this and issue #117, but separately. Both issues should absolutely be done to increase user experience. Due to issue #117 there are users that do not recognize the changes within the form. When reading issue #117 description, looks like we need to figure out what is the best solution to resolve the challenge.  Anyway, I would prioritise only this issue before 01.02.23. :)
